easybuild/tools/filetools.py

```diff
@@ -1180,16 +1225,12 @@ def move_logs(src_logfile, target_logfile):
 
             # retain old logs
             if os.path.exists(new_log_path):
-                i = 0
-                oldlog_backup = "%s_%d" % (new_log_path, i)
-                while os.path.exists(oldlog_backup):
-                    i += 1
-                    oldlog_backup = "%s_%d" % (new_log_path, i)
-                shutil.move(new_log_path, oldlog_backup)
-                _log.info("Moved existing log file %s to %s" % (new_log_path, oldlog_backup))
+                back_up_file(new_log_path)
 
+            # remove first to ensure portability (shutil.move might fail when overwriting files in some systems)
+            remove_file(new_log_path)
```

Comment:
this should be part of `move_file` (under the condition that the target location already exists)?